src/display/annotation_layer.js

```diff
@@ -315,6 +335,16 @@ var TextAnnotationElement = (function TextAnnotationElementClosure() {
 
   Util.inherit(TextAnnotationElement, AnnotationElement, {
     /**
+     * Check does this annotation need HTML element.
+     *
+     * @public
+     * @memberof TextAnnotationElement
+     */
+    hasHtml: function TextAnnotationElement_hasHtml() {
+      return this.data.hasPopup || this.data.title || this.data.contents;
```

Comment:
Looking through the code slightly more in depth, it appears this would probably work. So you can probably disregard the comment.